src/test/java/Homework23.java

```diff
@@ -0,0 +1,34 @@
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import pages.BasePage;
+import pages.HomePage;
+import pages.LoginPage;
+import org.openqa.selenium.WebDriver;
+
+public class Homework23 extends BasePage {
+
+    public Homework23(WebDriver givenDriver) {
+        super(givenDriver);
+    }
+
+    @Test
+
+    public void renamePlaylist()throws InterruptedException{
```

Comment:
Please, delete `throws InterruptedException`

**Thread. sleep is bad! It blocks the current thread and renders it unusable for further work.
Using Thread. sleep() frequently in an automation framework is not a good practice.**

Please use the Explicit Wait command, the WebDriver is directed to wait until a certain condition occurs before proceeding with executing the code.